README.md

```diff
@@ -131,6 +132,30 @@ export class Shout extends Component<ShoutSignature> {
 
 [yieldable named blocks rfc]: https://github.com/emberjs/rfcs/blob/master/text/0460-yieldable-named-blocks.md
 
+#### Template Components
+
+In GlimmerX, standalone templates can be invoked as components. Like class-based components, you can declare a signature for a template component in order for Glint to understand how it can be used.
+
+The class-based component above could as a template component like so:
+
+```ts
+import { TemplateComponent } from '@glint/environment-glimmerx/component';
+
+interface ShoutSignature {
+  /* same as above */
+}
+
+const shout = (message: string) => `${message.toUpperCase()}!`;
+
+const Shout: TemplateComponent<ShoutSignature> = hbs`
+  <div ...attributes>
+    {{yield (shout @message)}}
+  </div>
+`;
+```
+
+Note that, similar to React's `FunctionComponent` and `FC`, you can also import and use the `TC` type alias as a shorthand for `TemplateComponent`.
+
```

Comment:
Suggested change (can't use an actual suggestion because nested code blocks like that aren't a thing!):

> Note that, similar to React's `FunctionComponent` and `FC`, you can also import and use the `TC` type alias as a shorthand for `TemplateComponent`:
> 
> ```ts
> import type { TC } from '@glint/environment-glimmerx/component';
> ```